### DISCUSSION TITLE
is there a way to get a dependency graph listing from my manifest?

### DISCUSSION THREAD

**Discussion by willie** [microsoft/vcpkg#37464] [Q&A] [answered]
I recently went all-in on vcpkg, relying on it more than ever in a project.

Sadly, one of my new dependencies decided to rely upon the `python3` port, which doesn't build on `linux/arm64` at the moment (issue #37451). I could back out of a dependency to remove the need to compile `python3`, if I only knew which one was indirectly using it. 

Is there a vcpkg call or a script that would ferret this out? I searched but found nothing.

**Comment by willie** [comment, marked as answer]
Clearly, I need sleep. https://learn.microsoft.com/en-us/vcpkg/commands/depend-info solves this.

**Reply by dg0yt** [reply]
Sometimes, `vcpkg install --dry-run <list>` is useful. It lets you change features for multiple ports. And it shows the triplet, which may matter for complex host dependencies.
